### PR TITLE
Simple API client and issue-details component

### PR DESCRIPTION
**Pull request by djdupre**
These changes add the issue service to pull data from our non-existent API.
Also added is the issue-details component which demonstrates the usage of this service.

Sorry if this is kind of a big pull request.

### CODE REVIEW

**Comment by cjteam123** [comment]
Looks like you need to fix some conflicts first

File: src/app/issue.service.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
```
Comment (cjteam123) [inline src/app/issue.service.ts]: Could you match the import styles up please

Comment (cjteam123) [reply src/app/issue.service.ts]: There are a mix of styles throughout the files

Comment (djdupre) [reply src/app/issue.service.ts]: Fixed by d03d1873c0aba72e6776df6be3258249f86e5153

**Pull request merged**